test/checkpoint/test_checkpoint.c

```diff
@@ -41,22 +40,24 @@ static int  wt_shutdown(void);
 
 extern int __wt_optind;
 extern char *__wt_optarg;
+static char home[512];
```

Comment:
You should probably use the ```home``` defined in the global structure defined in ```test_checkpoint.h```, rather the declaring a new one here.

You can change that definition to be an array, rather than a ```char *```